monailabel/transform/writer.py

```diff
@@ -58,6 +60,85 @@ def write_itk(image_np, output_file, affine, dtype, compress):
     itk.imwrite(result_image, output_file, compress)
 
 
+def write_seg_nrrd(
+    image_np: np.ndarray,
+    output_file: str,
+    dtype: type,
```

Comment:
may be u can have default values for dtype

Reply:
We tried to make this function's signature similar to the `write_itk()`. It is probably best to also have a default there as well then, would you agree?